specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/examples/AgentPoolsCreate_Flex.json

```diff
@@ -0,0 +1,155 @@
+{
+  "parameters": {
+    "api-version": "2023-08-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "agentPoolName": "agentpool1",
+    "parameters": {
+      "properties": {
+        "orchestratorVersion": "",
```

Comment:
minor: Is this field a required input or something which is why it's specified + empty here?

Seems sorta odd, during create if you want us to pick this wouldn't you just omit it entirely?

Reply:
You can leave it empty and we'll use the default for create.